libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -314,7 +314,7 @@ - (void)testCreateQuerySearchDelete {
 // - query updated object (requestForSearch) and make sure we don't get anything
 - (void)testCreateUpdateQuerySearchDelete {
     // create object
-    NSString *lastName = [NSString stringWithFormat:@"Doe-%@", [NSDate date]];
```

Comment:
@khawkins so an alternative fix can be made on line 334 to correctly escape the query, but it's just another way to fix test issue. It's not a real bug..